### ISSUE TITLE
Properties for configuration files are broken

### ISSUE THREAD

**Issue by rwalravens** [jshiell/checkstyle-idea#565]
Since 5.64.0 properties are broken.

When a configuration file is set as active, the following error occurs:
```
The Checkstyle rules file could not be parsed.
Property ${............} has not been set
The file has been blocked for 60s.
```

This was configured for my project (checkstyle-idea.xml):
```
<entry key="active-configuration" value="LOCAL_FILE:D:/projects/checkstyle.xml:CBSS" />
<entry key="checkstyle-version" value="8.33" />
<entry key="copy-libs" value="true" />
<entry key="location-0" value="BUNDLED:(bundled):Sun Checks" />
<entry key="location-1" value="BUNDLED:(bundled):Google Checks" />
<entry key="location-2" value="LOCAL_FILE:D:/projects/checkstyle.xml:CBSS" />
<entry key="property-2.checkstyle.suppressions.file" value="D:\projects\checkstyle-suppressions.xml" />
<entry key="scan-before-checkin" value="false" />
<entry key="scanscope" value="JavaOnlyWithTests" />
<entry key="suppress-errors" value="false" />
```
Since 5.64.0, in the settings screen, the value of property-2 is not shown anymore, and when altered, it is not serialized anymore.

**Comment by jshiell** [comment]
Ah bugger, that part was touched by the pull request in 5.64. I’ll withdraw the release until I can have a look - thanks very much for letting me know!

**Comment by jshiell** [comment]
A good start at least - I can reproduce this. It appears to be related to active configurations only - the values are set correctly from the properties window, but if you quite and restart then they are not loaded. If the configurations are inactive then they _are_ loaded.

**Comment by jshiell** [comment]
Fixed in 5.65.0.
